src/page-modules/assistant/assistant.module.css

```diff
@@ -50,10 +50,12 @@
   grid-area: alternatives;
   width: 100%;
   background-color: var(--static-background-background_accent_1-background);
-  overflow: hidden;
```

Comment:
This was here because of the animation. The collapsing/uncollapsing makes the content overflow the container and not look too great. Is it necessary to remove this?

Reply:
overflow:hidden was removed because the content from the via search input field was overlaped by the trip results as shown in the image. When removed, the content overlaps the trip results instead. 

<details>

<img width="587" alt="Screenshot 2024-02-07 at 20 57 58" src="https://github.com/AtB-AS/planner-web/assets/59939294/759b0ae3-d4d6-4357-ae9a-bee36b12017b">

<img width="1008" alt="Screenshot 2024-02-08 at 08 28 43" src="https://github.com/AtB-AS/planner-web/assets/59939294/e4866378-efbe-4045-869d-5398bd90ab0b">

</details>

Reply:
I see. What we can do is to add temporary CSS attributes with framer-motion, so that when collapsing happens the container does have `overflow: hidden;` to avoid the contents spilling out of it. Can be achieved by using `transitionEnd`, like this: 

![2024-02-08 at 14 13 27](https://github.com/AtB-AS/planner-web/assets/14045515/80f31639-b419-4554-9bf5-3946efa75158)


But not sure if it's a big deal. I guess most users barely notice it anyway.